### PR TITLE
[ACA-3793] - Make e2e download folder path OS agnostic

### PR DESCRIPTION
**Pull request by KikoUK**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe: e2e runtime fix

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/ACA-3793

**What is the new behaviour?**

https://issues.alfresco.com/jira/browse/ACA-3793

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: protractor.conf.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ require('dotenv').config({ path: process.env.ENV_FILE });
 
 const SmartRunner = require('protractor-smartrunner');
 const projectRoot = path.resolve(__dirname);
-const downloadFolder = `${projectRoot}/e2e-downloads`;
+const downloadFolder = path.join(__dirname, '/e2e-downloads');
```
Comment (DenysVuika) [inline protractor.conf.js:14]: this change does not make any sense unless you remove '/' (non-windows) separator from the folder name. The 'path.jon' is automatically adding separators. But why we need this change anyway?

Comment (KikoUK) [reply protractor.conf.js:14]: Regarding the first question - I've added the forward slash separator to be consistent with ADF protractor.conf.js e.g.
image
Double forward slashes in paths also work on Windows

Regarding the second question -- I've raised this ticket as we have 3 people in our team running e2e locally on Windows machines and the suggested fix should make e2e run on any OS without failures.

Comment (DenysVuika) [inline protractor.conf.js:14]:
```suggestion
const downloadFolder = path.join(__dirname, 'e2e-downloads');
```

**Comment by KikoUK** [comment]
> added the suggestion for the path.json, there are also other cases of slashes in the file, like
> 
> ```
> var filePath = dirPath + '/' + files[i];
> ```
> 
> is that working on windows for you?

Yeah, I've not seen a related error

**Pull request merged**